woocommerce-payments.php

```diff
@@ -12,7 +12,7 @@
  * WC tested up to: 6.1
  * Requires at least: 5.6
  * Requires PHP: 7.0
- * Version: 3.5.0
+ * Version: 3.6.0
```

Comment:
Do we need to commit this change as part of this PR?

Reply:
Yes good point, it should be already taken care of in other PR

Reply:
Reverted